lib/chef/request_id.rb

```diff
@@ -15,7 +15,6 @@
 # limitations under the License.
 #
 
-require 'securerandom'
```

Comment:
Learning opportunity for me: how can you tell it is in the standard library?

Looking at http://ruby-doc.org/core-2.2.0/Random.html it seems the only difference from http://ruby-doc.org/stdlib-2.2.0.preview1/libdoc/securerandom/rdoc/SecureRandom.html is that `Secure` is in core, while `SecureRandom` is in stdlib.  Do we not need to require things in core?